File: DSA(Basic)/Arrays/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
       vector<int> ans{0,0};
       int size = nums.size();
       for(int i = 0;i<size;i++){
           for(int j =i+1;j<size;j++)
           {
               if(nums[i]  + nums[j] == target)
                   {
                      ans[0]=i;
                      ans[1]=j;
                      
                   }
                   
           }
            
       }
        return ans;
    }
     
};

File: DSA(Basic)/Arrays/TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> hm = new HashMap<Integer,Integer>();
        int res[] = new int[2];
        int j=0;
        for(int i=0;i<nums.length;i++){
            if(hm.containsKey(target-nums[i])){
                 res[0] = hm.get(target-nums[i]);
                 res[1] = i;
            }
              hm.put(nums[i],i);
            
        }
      return res;
      
    }
}
